schema.ts

```diff
@@ -31,13 +31,19 @@ export interface ModuleDoc {
   exports?: Array<ExportDoc>;
 }
 
-export type ExportDoc = ClassDoc|FunctionDoc|VariableDoc;
+export type ExportDoc = ClassDoc | FunctionDoc | VariableDoc | DefinitionDoc;
+
+export interface DefinitionDoc {
```

Comment:
I know it's long, but we should probable call this `CustomElementDefinitionDoc`.